survey.html

```diff
@@ -63,61 +63,64 @@ <h1>TolakedCFC Survey Form</h1>
                 <option value="other">Other</option>
             </select>
             </div>
+            
                         <!-- RADIO OPTION -->
             <div class="details">
 
-                <label for="radio">Would you recommend TolakedCFC to a friend?</label>
+                    <label for="radio">Would you recommend TolakedCFC to a friend?</label>
                 
-                <label for="recommend-1">
-                    <input type="radio" name="recommend" id="recommend-1"> Yes</input>
-                </label>
+                <div>
+                    <label for="recommend-1"> <input type="radio" name="recommend" id="recommend-1"> Yes</input></label>
```

Comment:
Don't embed the input tag in label tag. Also input is a self-closing tag, it does not require closing tag.